fastdupes.py

```diff
@@ -439,14 +439,14 @@ def pruneUI(dupeList, mainPos=1, mainLen=1):
     :rtype: :class:`~__builtins__.int`
     """
     dupeList = sorted(dupeList)
-    print
+    print('')
     for pos, val in enumerate(dupeList):
-        print "%d) %s" % (pos + 1, val)
```

Comment:
This line changes the output format from "1) name" to "1 name". Please restore the original behaviour.